aspnetcore/release-notes/aspnetcore-8.0.md

```diff
@@ -377,37 +377,37 @@ builder.On<string, string>("ReceiveMessage", (user, message) => ...
 await builder.StartAsync();
 ```
 
-### SignalR seamless reconnect
+### SignalR stateful reconnect
 
-SignalR seamless reconnect reduces the perceived downtime of clients that have a temporary disconnect in their network connection, such as when switching network connections or a short temporary loss in access.
+SignalR stateful reconnect reduces the perceived downtime of clients that have a temporary disconnect in their network connection, such as when switching network connections or a short temporary loss in access.
 
-Seamless reconnect achieves this by:
+Stateful reconnect achieves this by:
 
 * Temporarily buffering data on the server and client.
 * Acknowledging messages received (ACK-ing) by both the server and client.
 * Recognizing when a connection is returning and replaying messages that may have been sent while the connection was down.
 
 To opt-in to seamless reconnect:
 
-* Update the .NET client code to enable the `UseAcks` option:
+* Update the JavaScript or TypeScript client code to enable the `withStatefulReconnect` option:
```

Comment:
Should we also show the .NET client?

Reply:
Yikes!  Yes, sorry Brennan, I was focusing on JavaScript and TypeScript and left out the .NET Client.   I will add it back in!